api/src/main/java/org/openmrs/util/MemoryAppender.java

```diff
@@ -44,7 +44,7 @@
 	// we store the MemoryAppenders by name, using SoftReferences to allow them to be garbage collected
 	// as an implementation detail, we expect this class to only have a single instance, so our map
 	// is only allocated an initial capacity of 1
-	private final static Map<String, SoftReference<MemoryAppender>> APPENDERS = new HashMap<>(1);
+private static final  Map<String, SoftReference<MemoryAppender>> APPENDERS = new HashMap<>(1);
```

Comment:
There is also no tab here, in addition, there are probably two spaces between the words "final" and "Map". ;)

Reply:
> There is also no tab here, in addition, there are probably two spaces between the words "final" and "Map". ;)

hello @insookwa , 
have you seen these comments?

Reply:
please @insookwa  can you squash your commit? 
Thanks

Reply:
@achilep did you get a chance to look at this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

Reply:
thanks @dkayiwa  i am still finding ways of squashing commits . should I just delete this pull request and commit another one with the proper changes.

Reply:
Did you read the link that i shared?

Reply:
Actually the link discourages squashing which I've been trying to figure out. I am very sorry   . is there a way I can fix this best @dkayiwa

Reply:
![image](https://user-images.githubusercontent.com/9057882/92354287-535b7800-f0e2-11ea-8482-24939db93011.png)
Here, the tab at the beginning is still missing and there are two spaces between the words "final" and "Map". ;)

Reply:
If we take several adjacent lines, the lack of tabs will be better seen:
![image](https://user-images.githubusercontent.com/9057882/92354619-f44a3300-f0e2-11ea-9b36-be4216ae9d76.png)

Reply:
thanks @dkayiwa  for this information .

> @achilep did you get a chance to look at this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482